PowerShellTools.HostService/ServiceManagement/Debugging/PowerShellDebuggingServiceEventHandlers.cs

```diff
@@ -116,14 +126,31 @@ private void Debugger_DebuggerStop(object sender, DebuggerStopEventArgs e)
                         file = _mapRemoteToLocal[bp.Script];
                     }
 
-                    _callback.DebuggerStopped(new DebuggerStoppedEventArgs(file, bp.Line, bp.Column));
+                    _callback.DebuggerStopped(new DebuggerStoppedEventArgs(true, file, bp.Line, bp.Column, false));
                 }
             }
             else
-            {
+            {                
                 if (_callback != null)
                 {
-                    _callback.DebuggerStopped(new DebuggerStoppedEventArgs());
+                    if (_attaching)

```

Comment:
Is there any other way you can use from runspace itself in order to determine this is "attaching"? e.g named pipeline runspace? Use this kind of flag would be my last choice considering its fragility, not thread safe, also can get out of sync in some error cases which would cause unexpected result.